apps/backend/src/modules/users/user.repository.ts

```diff
@@ -74,12 +116,15 @@ class UserRepository implements Repository {
 		const user = await this.userModel
 			.query()
 			.withGraphFetched(
-				`[${RelationName.USER_DETAILS}, ${RelationName.USER_TASK_DAYS}]`,
+				`[${RelationName.USER_DETAILS}.[${RelationName.AVATAR}], ${RelationName.USER_TASK_DAYS}]`,
 			)
-			.findById(id);
+			.findById(id)
+			.castTo<null | UserWithAvatarFile>();
```

Comment:
```suggestion
			.castTo<undefined | UserWithAvatarFile>();
```
Don't use invalid casting, it can't be null there